docs/configuration.asciidoc

```diff
@@ -933,3 +933,33 @@ During startup the Node.js agent queries the local environment to determine whet
 - `none`: Do not query for any cloud provider information.
 
 If the value is not one of the five listed above, the agent will use the value of `auto`.
+
+[[ignore_message_queues]]
```

Comment:
```suggestion
[[ignore-message-queues]]
```

How do you feel about dashes here to maintain consistency with the other anchors?

Reply:
👍 -- making this change.  @bmorelli25 are there automated docs linting tools that would catch this sort of thing for us?